content/pt/docs/guias/integracao-para-fornecedores/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "INTEGRAÇÃO INTERNA - PANDA"
-linkTitle: "INTEGRAÇÃO INTERNA - PANDA"
+title: "INTEGRAÇÃO INTERNA - PANDA (Draft)"
+linkTitle: "INTEGRAÇÃO INTERNA - PANDA (Draft)"
```

Comment:
those draft contents shouldn't be with the draft: true frontmatter flag? or they'll be in production like this?

Reply:
but they are with draft:true, no?